utils/spacewalk-common-channels.ini

```diff
@@ -548,15 +561,105 @@ checksum = sha256
 base_channels = opensuse_leap15_0-%(arch)s
 yumrepo_url = http://download.opensuse.org/update/leap/15.0/non-oss/
 
-[spacewalk28-client-opensuse_leap_15_0]
-name     = Spacewalk Client 2.8 %(base_channel_name)s
+[opensuse_leap15_0-uyuni-client]
+name     = Uyuni Client Tools for %(base_channel_name)s
 archs    = x86_64
 base_channels = opensuse_leap15_0-%(arch)s
 checksum = sha256
-gpgkey_url = http://download.opensuse.org/repositories/systemsmanagement:/spacewalk:/2.8/openSUSE_Leap_15.0/repodata/repomd.xml.key
-gpgkey_id = A0E46E11
-gpgkey_fingerprint = 68D3 3874 9967 0AEB D988  2DB3 2ABF A143 A0E4 6E11
-yumrepo_url = http://download.opensuse.org/repositories/systemsmanagement:/spacewalk:/2.8/openSUSE_Leap_15.0/
+gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/openSUSE_Leap_15-Uyuni-Client-Tools/openSUSE_Leap_15.0/repodata/repomd.xml.key
+gpgkey_id = %(_uyuni_client_gpgkey_id)s
+gpgkey_fingerprint = %(_uyuni_client_gpgkey_fingerprint)s
+yumrepo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/openSUSE_Leap_15-Uyuni-Client-Tools/openSUSE_Leap_15.0/
+
+[opensuse_leap15_0-uyuni-client-devel]
+name     = Uyuni Client Tools for %(base_channel_name)s (Development)
+archs    = x86_64
+base_channels = opensuse_leap15_0-%(arch)s
+checksum = sha256
+gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/openSUSE_Leap_15-Uyuni-Client-Tools/openSUSE_Leap_15.0/repodata/repomd.xml.key
+gpgkey_id = %(_uyuni_client_gpgkey_id)s
+gpgkey_fingerprint = %(_uyuni_client_gpgkey_fingerprint)s
+yumrepo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/openSUSE_Leap_15-Uyuni-Client-Tools/openSUSE_Leap_15.0/
+
+[SUSE_12_SP3-uyuni-client]
```

Comment:
Hmm - maybe it would be better to use "SLE" term here.

Reply:
Agreed, will change it as well.

Reply:
All changed to sleXX-spY and sleXX.